classes/install/Upgrade.inc.php

```diff
@@ -1577,6 +1582,10 @@ function convertCommentsToEditor() {
 			$comments_to_ed = PKPString::stripUnsafeHtml($row['comments_to_ed']);
 			
 			if ($comments_to_ed != ""){
+				
+				$user_id = $row['user_id'];
+				if (!$user_id) $user_id = 1;
```

Comment:
Yes, I think this should also always be the case -- this stage_assignment is migrated correctly... Thus maybe you can also totally remove it...